### PR TITLE
Updated files for the v4.0.23 maintenance release

### PR DESCRIPTION
**Pull request by TTVXStarWake**

### CODE REVIEW

**Comment by LaurentGoderre** [comment]
DTM! You got some markup issues in your HTML

**Comment by MaximMartel** [comment]
yes, fixing it right now, there is a section tag closing that i missed.

**Comment by MaximMartel** [comment]
so im Dead to You @LaurentGoderre  (DTM = Dead to me) lol

**Comment by MaximMartel** [comment]
@LaurentGoderre Just the saucelabs timeout, you want to merge this one ?

**Comment by duboisp** [comment]
@MaximMartel you know the wet-boew master are not ready yet for the next release, In the current master version there is WCAG contrasting issue and bugs that has been identified and fix submitted through PR that have not been merged yet.

**Comment by MaximMartel** [comment]
Thanks for flagging this up @duboisp !

**Comment by LaurentGoderre** [comment]
@MaximMartel I meant DNM (do not merge)

**Comment by MaximMartel** [comment]
@LaurentGoderre i know, it was just a joke. 

I will need to merge in #7740 before closing this release.

**Comment by LaurentGoderre** [comment]
That PR needs to be updated before though

**Comment by MaximMartel** [comment]
I will need to push the PR in, then test it before releasing, then update all the files on that PR , amend my changes... right?

**Comment by LaurentGoderre** [comment]
The pr is not quite ready but when it is, you'll merge, then you can just modify your release note by hand by copying the release message and bumping the commit by @duboisp by 1

**Comment by LaurentGoderre** [comment]
This will probably have to wait until tomorrow

**Comment by MaximMartel** [comment]
The 4.0.23 release will happen tommorow as we need to get a PR in, tested properly, then i can close 4.0.22.

**Comment by EricDunsworth** [comment]
Would it also be possible to merge #7683 before launching 4.0.23 (if everyone's on board with its changes)? It fixes WCAG 2.0 issues in all carousel implementations.

On a side note, there are currently 22 open pull requests in wet-boew (and others in theme/variant repos). Some are almost 2 years old. Is there any chance they could be reviewed over the long term? IMO they should be either merged in, closed, or discussed/updated if necessary. It's likely to discourage potential contributors if it looks like PRs are lingering around for too long.

**Comment by MaximMartel** [comment]
I will be closing the release at noon today.

**Comment by MaximMartel** [comment]
@EricDunsworth #7683 will be in the next release as I didn't had time to test prior to this release.

File: site/pages/docs/versions/rdmp-en.hbs [hunk 1]
```diff
@@ -46,6 +42,10 @@
 		</thead>
 		<tbody>
 			<tr>
+				<td><time>2016-11-16</time></td>
```
Comment (duboisp) [inline site/pages/docs/versions/rdmp-en.hbs]: I think it should be changed to 2016-11-18

File: site/pages/docs/versions/rdmp-fr.hbs [hunk 2]
```diff
@@ -46,6 +42,10 @@
 		</thead>
 		<tbody>
 			<tr>
+				<td><time>2016-11-16</time></td>
```
Comment (duboisp) [inline site/pages/docs/versions/rdmp-fr.hbs]: I think it should be changed to 2016-11-18

**Review by duboisp — CHANGES_REQUESTED** [review]

**Review by duboisp — APPROVED** [review]

**Comment by LaurentGoderre** [comment]
Rebase this please

**Comment by MaximMartel** [comment]
rebase done @LaurentGoderre

**Comment by MaximMartel** [comment]
only theme Government of Canada Web Usability is not available for download. Should be available very soon, sorry for any inconvenience that may cause. We are working toward fixing this issue.

**Comment by MaximMartel** [comment]
all theme closed, release 4.0.23 is now officially completed.

**Pull request merged**